sdk/typescript/examples/plain-html/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import { createNymMixnetClient, NymMixnetClient } from '@nymproject/sdk';
+import { displayReceived, sendMessageTo, displaySenderAddress } from './dom-utils';
+
+let nym: NymMixnetClient | null = null;
+
+/**
+ * The main entry point
+ */
+async function main() {
+  nym = await createNymMixnetClient();
+
+  if (!nym) {
+    console.error('Oh no! Could not create client');
+    return;
+  }
+
+  // // mixnet v1
+  // const validatorApiUrl = 'https://validator.nymtech.net/api';
+  // const preferredGatewayIdentityKey = 'E3mvZTHQCdBvhfr178Swx9g4QG3kkRUun7YnToLMcMbM';
+
+  // mixnet v2
+  const validatorApiUrl = 'https://qwerty-validator-api.qa.nymte.ch/api'; // "http://localhost:8081";
+  const preferredGatewayIdentityKey = undefined; // '36vfvEyBzo5cWEFbnP7fqgY39kFw9PQhvwzbispeNaxL';
```

Comment:
if we're not even using it, why not remove it

Reply:
mmmm, I really want to remove all of that and make it just work with `gateway1.nymtech.net` on mainnet